src/init.cpp

```diff
@@ -553,6 +553,7 @@ void SetupServerArgs(ArgsManager& argsman)
     argsman.AddArg("-bytespersigop", strprintf("Equivalent bytes per sigop in transactions for relay and mining (default: %u)", DEFAULT_BYTES_PER_SIGOP), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
     argsman.AddArg("-datacarrier", strprintf("Relay and mine data carrier transactions (default: %u)", DEFAULT_ACCEPT_DATACARRIER), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
     argsman.AddArg("-datacarriersize", strprintf("Maximum size of data in data carrier transactions we relay and mine (default: %u)", MAX_OP_RETURN_RELAY), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
+    argsman.AddArg("-fullrbf", strprintf("Accept transaction replacement without opt-in replaceability inspection (default: %u)", DEFAULT_TRANSACTION_REPLACEMENT), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
```

Comment:
Concept NACK on a new option. The `-mempoolreplacement` option was designed for this use case, and has been long-supported for it in Knots.

Reply:
It was removed in https://github.com/bitcoin/bitcoin/pull/16171

Reply:
It's a valid discussion how the option should be called but NACKing this based on disagreement on the option name is silly imo.

Reply:
> 0: Disable RBF

I think this is getting a bit out of hand to re-add a largely unused setting that was removed with overall support, see also https://github.com/bitcoin/bitcoin/pull/16171#issuecomment-501303451

No opinion on how the new setting should be named.

Reply:
Right, if people want to shedpaint names perhaps open a separate issue to do so. I don't think that needs to be discussed on this PR.

Reply:
@1440000bytes: The suggestion of reversing #16171 would be a (new) separate PR. (It was merged 3 years ago.) A rename discussion for this particular PR would be a (new) issue.

Reply:
I have seen zero desire from users to revert the removal of the older option. Assuming we don't want to support features no one is asking for, it also means we should pick an argument name that hasn't existed before to remove ambiguity.

Reply:
`-mempoolreplacement=-optin` is non-ambiguous.

Reply:
Right, we can get even more specific perhaps. I don't personally care too much.

Reply:
Marking this as resolved as the conversation was just restarted down-thread: https://github.com/bitcoin/bitcoin/pull/25353#discussion_r910203543.